src/nl/hannahsten/texifyidea/highlighting/LatexAnnotator.kt

```diff
@@ -213,11 +211,32 @@ open class LatexAnnotator : Annotator {
     /**
      * Annotates the contents of the given parameter with the given style.
      */
+    @Suppress("USELESS_CAST")
     private fun AnnotationHolder.annotateRequiredParameter(parameter: LatexRequiredParam, style: TextAttributesKey) {
-        val content = parameter.firstChildOfType(LatexContent::class) ?: return
-        this.newSilentAnnotation(HighlightSeverity.INFORMATION)
-            .range(content)
-            .textAttributes(style)
-            .create()
+        val firstContentChild = parameter.firstChildOfType(LatexContent::class)
+        val firstParamChild = parameter.firstChildOfType(LatexRequiredParamContent::class)
+
+        if (firstContentChild != null) {
+            this.newSilentAnnotation(HighlightSeverity.INFORMATION)
+                .range(firstContentChild)
+                .textAttributes(style)
+                .create()
+        }
+        else if (firstParamChild != null) {
+            parameter.childrenOfType(LeafPsiElement::class)
+                // Take out only NORMAL_TEXT_WORD
+                .filter {
+                    it.elementType.index == 195.toShort()
```

Comment:
This number may change at any time I believe, this won't work. Why not do `childrenOfType(LatexNormalText::class)`? Though I'm not sure why only the text, as far as I'm concerned you can make the RequiredParamContent all styled.

Reply:
`RequiredParamContent` would make nested commands formatted as well. I only want plaintext to take up the format

Reply:
`childrenOfType(LatexNormalText::class)` returns no results, `RequiredParamContent` returns too much

`\textbf{BOLD \textit{italics} }`
![image](https://user-images.githubusercontent.com/66704796/201414297-ae7edd87-e20d-4e92-92f4-a2a03baad317.png)
results in 
![image](https://user-images.githubusercontent.com/66704796/201414450-78816484-3375-462e-adb3-265fdba0c71c.png)